node/impl/eth/api.go

```diff
@@ -25,6 +28,143 @@ var (
 
 var log = logging.Logger("node/eth")
 
+// EthFilecoin -------------------------------------------------------------------------------------
```

Comment:
everything in here except `TipSetResolver` is just moving these interfaces out of their respective impl files to here, not a strictly necessary change, just came about during my circular refactoring journey trying to improve things; unclear whether I actually did improve things though!